recipes-extended/ofdpa-tools/ofdpa-tools/switch_tcpdump

```diff
@@ -38,15 +38,18 @@ import time
 EPILOG = """
 Example usage:
 
-Capture ingress packets on port3 and write at most 100 MB to /tmp/capture.pcap:
-switch_tcpdump --inPort port3 --filePath /tmp/capture.pcap --maxSize 100
+Capture ingress packets on port3 and write at most 100 MB to /tmp/pcap:
+switch_tcpdump --inPort port3 --filePath /tmp/pcap
```

Comment:
The wording makes this sound like this is an explicit limit (as it was before), now it is an implicit limit. So I'd rather write it

```suggestion
Capture ingress packets on port3 and write to /tmp/pcap (will write at most 100 MB):
switch_tcpdump --inPort port3 --filePath /tmp/pcap
```

to make it more clear that it is the side effect of not specifying a limit.

Also you are later changing 'arriving' to 'seen' for the extra arguments (assuming you want to imply that you also see packets sent by the controller/OS), but here it still has 'ingress packets'. You might want to update this as well. Maybe just use 'packets seen' as well for consistent language.

Finally, please explain your reasoning for changes (especially the 'arriving' to 'seen') in the commit message and don't leave it empty.

Reply:
I rephrased this part and removed the misleading usage of "ingress".